app/views/layouts/_navbar.html.erb

```diff
@@ -17,6 +17,18 @@
       <li class="nav-item">
         <%= link_to 'Contact', contact_path, class: 'nav-link' %>
       </li>
+      <li>
+      <%= link_to "Opportunities", opportunities_path, class: 'nav-link' %>
+      </li>
+      <li>
+      <%= link_to "Slack", slack_path, class: 'nav-link'%>
+      </li>
+      <li>
+      <%= link_to "Getting Started", gs_path, class: 'nav-link'%>
+      </li>
+      <li>
+      <%= link_to "Learn Ruby on Rails", learn_ror_path, class: 'nav-link' %>
+      </li>
     </ul>
     <ul class="navbar-nav">
       <li class="nav-item">
```

Comment:
Can you fix the indentation?

It should look like:
```html
<ul>
      <li>
          <%= link_to "Getting Started", gs_path, class: 'nav-link'%>
      </li>
      <li>
          <%= link_to "Learn Ruby on Rails", learn_ror_path, class: 'nav-link' %>
      </li>
</ul>
```

4 spaces for html, not two (get me to help you setting VS Code to recognise erb and HTML files)